tests/kafkatest/services/performance/templates/tools_log4j.properties

```diff
@@ -19,7 +19,7 @@ log4j.rootLogger = {{ log_level|default("INFO") }}, FILE
 log4j.appender.FILE=org.apache.log4j.FileAppender
 log4j.appender.FILE.File={{ log_file }}
 log4j.appender.FILE.ImmediateFlush=true
-# Set the append to false, overwrite
-log4j.appender.FILE.Append=false
+# Set the append to true, append
+log4j.appender.FILE.Append=true
```

Comment:
I may have made this change in the original branch back in December, and I honestly don't remember if I had a good reason for it.

I agree it's probably safer to revert this since several tools rely on it.